oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapAggregator.java

```diff
@@ -275,7 +276,9 @@ private MapAggregator(MapAggregator<U, ?> obj, MapReducer<IndexValuePair<U, X>>
    *
    * @param typeFilter the set of osm types to filter (e.g. `EnumSet.of(OSMType.WAY)`)
    * @return a modified copy of this object (can be used to chain multiple commands together)
+   * @deprecated use oshdb-filter {@link #filter(String)} instead
```

Comment:
Are these `deprecated` changes because of the integration of ohsome filter? Or are these deprecations a general recommendation?

Reply:
well, I would say that since the oshdb-filter is a more capable interface there's no need to drag around dozens of overloaded methods just for convenience.

But that's in principle up for discussion. In some situations it might be simpler to write `osmTag("key with spaces")` compared to `filter("\"key with spaces\" = *")` (note the quoting required by the spaces present in the tag key), but there are more like rare corner cases IMHO.

Reply:
The question was more meant if this deprecation is dependent on the *integration* of ohsome-filter as oshdb-filter, or if this could have been done anyway.